components/telemetry-operator/config/crd/bases/telemetry.kyma-project.io_logpipelines.yaml

```diff
@@ -0,0 +1,111 @@
+      openAPIV3Schema:
+        description: LogPipeline is the Schema for the logpipelines API
+        properties:
+          apiVersion:
+            description: 'APIVersion defines the versioned schema of this representation
+              of an object. Servers should convert recognized schemas to the latest
+              internal value, and may reject unrecognized values. More info: https://git.k8s.io/community/contributors/devel/sig-architecture/api-conventions.md#resources'
+            type: string
+          kind:
+            description: 'Kind is a string value representing the REST resource this
+              object represents. Servers may infer this from the endpoint the client
+              submits requests to. Cannot be updated. In CamelCase. More info: https://git.k8s.io/community/contributors/devel/sig-architecture/api-conventions.md#types-kinds'
+            type: string
+          metadata:
+            type: object
+          spec:
+            description: LogPipelineSpec defines the desired state of LogPipeline
+            properties:
+              files:
+                items:
+                  description: FileMount provides file content to be consumed by a
+                    LogPipeline configuration.
+                  properties:
+                    content:
+                      type: string
+                    name:
+                      type: string
+                  type: object
+                type: array
+              filters:
+                items:
+                  description: Filter describes a Fluent Bit filter configuration
+                    section.
+                  properties:
+                    content:
+                      type: string
+                  type: object
+                type: array
+              multilineParsers:
+                items:
+                  description: MultiLineParser describes a Fluent Bit multiline parser
+                    configuration section.
+                  properties:
+                    content:
+                      type: string
+                  type: object
+                type: array
+              outputs:
+                items:
+                  description: Output describes a Fluent Bit output configuration
+                    section.
+                  properties:
+                    content:
+                      type: string
+                  type: object
+                type: array
+              parsers:
+                items:
+                  description: Parser describes a Fluent Bit parser configuration
+                    section.
+                  properties:
+                    content:
+                      type: string
+                  type: object
+                type: array
+              secretRefs:
+                items:
+                  description: SecretReference is a pointer to a Kubernetes secret
+                    that should be provided as environment to Fluent Bit.
```

Comment:
yesss! correct relative pronoun, no comma. Excellent!

(still don't like splitting sentences per line tho, but at least here it's split between main clause and subordinate clause.)